src/components/Welcome.js

```diff
@@ -1,11 +1,18 @@
 import React from 'react';
+import PropTypes from 'prop-types';
 
-const Welcome = () => {
+const Welcome = (props) => {
+  const { onClick } = props;
```

Comment:
Nice destructuring, I like it. Another way to do this would be to destructure inline on line 4:

`const Welcome = ({onClick}) => {`

Reply:
Nice clean solution @laurenmbeatty !